frontend/src/slices/chat/chat.slice.ts

```diff
@@ -0,0 +1,66 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { MOCK_MESSAGE } from '#libs/constants/chat-mock-data.constant.js';
+import { FIRST_INDEX, LAST_INDEX } from '#libs/constants/index.constant.js';
```

Comment:
You have `#libs/constants/constants` file with reexports, take them from there

Reply:
Also I and @Xxandder moved some common constants to `shared/src/libs/constants` folder to make them accessible all over the project